docs: updating Wallets SDK docs to cover L1 Providers connection (#60)
immutable/imx-docs: docs/sdks/Wallets SDK Web.md

The build step is the starting point and is responsible for instantiating and providing the `WalletsSDK` object. To make it happen it is needed to import the `WalletsSDK` and call its static `build` method:

```ts
import {
  ENVIRONMENTS,
  WalletsSDK,
} from '@imtbl/imx-wallets-sdk-web';

> (async () => {
>   // Uses the static "build" function to get the SDK object instance
>   const walletsSdk = await WalletsSDK.build({ env: ENVIRONMENTS.STAGING });
> })();
> 
> (async () => {
>   // If you want to support WalletConnect, make sure to set your RPC config as well
>   // Reference: https://docs.walletconnect.com/quick-start/dapps/web3-provider#provider-options
>   const walletsSdk = await WalletsSDK.build({
>     env: ENVIRONMENTS.STAGING,
>     rpc: {
>       1: "https://mainnet.mycustomnode.com",
>     }
>   });
> })();

Reviewer: ```suggestion
(async () => {
  // RPC config is optional if you don't want to support WalletConnect
  // But If you do, make sure to set your RPC config following this reference:
  // Reference: https://docs.walletconnect.com/quick-start/dapps/web3-provider#provider-options
  const walletsSdk = await WalletsSDK.build({
    env: ENVIRONMENTS.STAGING,
    rpc: {
      1: "https://mainnet.mycustomnode.com",
    }
  });
})();
```

The connection step takes care of communicating with the selected L1 and L2 providers, establishing the connection, and retrieving the signers for each one of them, letting the L1 and L2 signers available in the `WalletConnection` object through the `getWalletConnection` method:
<!-- TODO: The WalletConnection is also returned by the connection function itself. Document it properly. -->

> **NOTICE:** Only the L1 provider is selectable at this point, the supported providers can be found on the enum `L1_PROVIDERS`. The L2 provider also has an enum (`L2_PROVIDERS`) but it is always resolved by default to `L2_PROVIDERS.IMX_WALLET` which is the SDK built-in L2 wallet solution. To check the available options for both L1 and L2 providers, access the [supported L1 wallets reference](#supported-l1-wallets) and [supported L2 wallets reference](#supported-l2-wallets).

> ##### MetaMask

Reviewer: In that case, I am not sure if we should have a build(init) session before talk about Metamask/WalletConnect connection?

```ts
import { 
import {
  ENVIRONMENTS,
  L1_PROVIDERS
  WalletsSDK, 
  L1_PROVIDERS,
  WalletsSDK,
} from '@imtbl/imx-wallets-sdk-web';

// Uses the static "build" function to get the SDK object instance
const walletsSdk = await WalletsSDK.build({ env: ENVIRONMENTS.STAGING });
async function connect() {
  // Uses the static "build" function to get the SDK object instance
  const walletsSdk = await WalletsSDK.build({ env: ENVIRONMENTS.STAGING });
  
  // Defines the connection params (L1 provider)
  const connectionParams = { provider: L1_PROVIDERS.METAMASK };
  // Connects to the desired wallets
  await walletsSdk.connect(connectionParams);
  
  // Gets the "WalletConnection" object which contains the L1 and L2 signers
  const walletConnection = walletsSdk.getWalletConnection();
}
```

// Defines the connection params (L1 provider)

> const connectionParams = { provider: L1_PROVIDERS.METAMASK };

Reviewer: instead of have a separate **walletconnect** and **metamask** section, can we just Comments here, say, the connectParam we can choose from 
L1_PROVIDERS.METAMASK -> will connect with metamask
L1_PROVIDERS.WALLETCONNECT -> will connect to walletconnect
